src/brogue/IO.c

```diff
@@ -4633,10 +4633,13 @@ short printMonsterInfo(creature *monst, short y, boolean dim, boolean highlight)
         }
     }
 
-    if (monst != &player
-        && (!(monst->info.flags & MONST_INANIMATE)
-            || monst->creatureState == MONSTER_ALLY)) {
+    if (y < ROWS - 1 && monst->wasNegated && monst->newPowerCount == monst->totalPowerCount
+        && (!player.status[STATUS_HALLUCINATING] || rogue.playbackOmniscience )) {
+        printString("      Negated       ", 0, y++, (dim ? &darkPink : &pink), &black, 0);
+    }
```

Comment:
Seems like this was duplicated out of the branch below - could you explain why it's in both places?

Reply:
I was lazy. Pushed an update that I think improves the overall flow.